subsys/bluetooth/host/hci_ecc.c

```diff
@@ -223,7 +223,7 @@ static void clear_ecc_events(struct net_buf *buf)
 {
 	struct bt_hci_cp_le_set_event_mask *cmd;
 
-	cmd = (void *)buf->data  + sizeof(struct bt_hci_cmd_hdr);
+	cmd = (void *)((u8_t *)buf->data  + sizeof(struct bt_hci_cmd_hdr));
```

Comment:
The `buf->data` is already `u8_t *` so this particular cast looks redundant. The added parenthesis behind the pointer increment is still needed as a fix though (I think).

Reply:
+1